src/components/Avatar.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const Avatar = ({ name, github, backgroundColor  }) => {
+  const initials = name.split(' ').map(n => n[0]).slice(0, 2).join('');
+
+  const handleClick = () => {
+    window.location.href = `https://github.com/${github}`
+  }
+
+  return (
+    <div
+    className="avatar"
+    style={{
```

Comment:
Move this to a css file. Dont use inline styling if you don't use the sx property for mui.